master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -445,7 +445,10 @@ public void updatePartitionSize() {
         Utils.bytesToString(tmpTotalWritten),
         tmpFileCount);
     if (tmpFileCount != 0) {
-      estimatedPartitionSize = tmpTotalWritten / tmpFileCount;
+      estimatedPartitionSize =
+          Math.max(
+              conf.minPartitionSizeToEstimate(),
+              Math.min(tmpTotalWritten / tmpFileCount, conf.maxPartitionSizeToEstimate()));
```

Comment:
Is there a case that estimatedPartitionSize uses `celeborn.master.estimatedPartitionSize.minSize` when `celeborn.master.estimatedPartitionSize.minSize` is greater than `celeborn.master.estimatedPartitionSize.maxSize`?

Reply:
> Is there a case that estimatedPartitionSize uses `celeborn.master.estimatedPartitionSize.minSize` when `celeborn.master.estimatedPartitionSize.minSize` is greater than `celeborn.master.estimatedPartitionSize.maxSize`?

That would be `celeborn.master.estimatedPartitionSize.maxSize`, IMO it's not a big problem. :) WDYT?